backend/uclapi/resources/views.py

```diff
@@ -0,0 +1,45 @@
+import os
+import requests
+import xml.etree.ElementTree
```

Comment:
`lxml` should be faster and easier to work with because the API is more Pythonic. Perhaps we should decide on one library to use globally in case we need to consume more XML (🤢 😷 🤧 🤒)  in the future?

Reply:
Also, there are some comments here on how to ensure that we process this data in a safe way: https://www.codacy.com/app/wilhelm.klopp/uclapi/pullRequest?prid=1025562

Reply:
I would say it depends whether we consider UCL trusted entity in the sense that they will not replace the availability XML with one of the bazilion XML attacks.

I would vote for `lxml` as well.

Reply:
I wouldn't worry about it too much at all since the data is coming from UCL. We aren't in anyway checking the data we get from databases, why would we do that for this? But I have changed it. I wouldnt be wrong if I say we are overkilling it sometimes :P